bundles/org.openhab.binding.lifx/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -45,4 +48,17 @@
 		</parameter>
 	</config-description>
 
+	<config-description uri="channel-type:lifx:effect">
+		<parameter name="effectMorphSpeed" type="decimal" min="0" max="25" step="0.1" required="false" unit="s">
+			<label>Morph effect speed</label>
+			<description>Speed of the morph effect in seconds.</description>
+			<default>3</default>
+		</parameter>
+		<parameter name="effectFlameSpeed" type="decimal" min="0" max="25" step="0.1" required="false" unit="s">
```

Comment:
In the [protocol](https://lan.developer.lifx.com/docs/firmware-effects#section-settileeffect-719) the `speed` is an unsigned 32-bit integer representing milliseconds. Wouldn't it make sense to increase the max value for users who want a really slow effect?